modules/chronograf-server/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "public_ip" {
+  value = "${aws_instance.chronograf_server.public_ip}"
+}
```

Comment:
You should also output:

* private_ip. In fact, many users won't want to run this in public subnets!
* instance_id
* iam_role_id